c2corg_ui/templates/whatsnew.html

```diff
@@ -0,0 +1,18 @@
+<%inherit file="base.html"/>
+<%namespace file="helpers/common.html" import="show_title"/>
+
+<%block name="metarobots">
+  <meta name="robots" content="index,follow">
```

Comment:
I don't think this page should be indexed. But no big deal.
Anyway this is already the default value: https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/base.html#L29
=> you can omit this block if you keep it as is.